riak/transports/pbc/connection.py

```diff
@@ -165,7 +165,7 @@ def _ssl_handshake(self):
 
                     return True
                 except ssl.SSLError as e:
-                    raise SecurityError(e.library + ": " + e.reason)
```

Comment:
Note to self: Always look at the code first.  Looks like 2.7.8 actually was the last version supporting PyOpenSSL.  Which is interesting.  I should tweak the versions in tox, then, too to reflect this.